.storybook/main.ts

```diff
@@ -0,0 +1,85 @@
+import type { StorybookConfig } from "@storybook/react-vite"
+import path, { join, dirname } from "path"
+import { mergeAlias } from "vite"
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
+ */
+function getAbsolutePath(value: string): any {
+  return dirname(require.resolve(join(value, "package.json")))
+}
+
+const config: StorybookConfig = {
+  stories: [
+    "../packages/**/*.mdx",
+    "../packages/**/*.stories.tsx",
```

Comment:
Hmm...so this is actually still the dev env for all packages then?
In that case, the dist stylesheets imported into the preview files will not be required given that Storybook will be building off the source code (would be nice to be able to test the dist styles, but can be considered a separate task).

Reply:
https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/4112122241/Idea+Introduce+a+production+flag+to+Stories